src/components/styles/Navbar.css

```diff
@@ -0,0 +1,58 @@
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 5rem;
+  box-shadow: -1px 2px 0 -1px #e8e8e8;
+  font-family: 'Montserrat', sans-serif;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+}
+
+.nav-content {
+  display: flex;
+  gap: 2rem;
+}
+
+.nav-content a {
+  text-decoration: none;
+  line-height: 6rem;
+}
+
+.bookstore-header {
+  font-size: 2rem;
+  color: rgb(41, 106, 180);
+}
+
+.books {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--black-two);
+  color: darkgrey;
+}
+
+.categories {
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: var(--black-two);
+  color: darkgrey;
+}
+
+.logo {
```

Comment:
- [ ] Your logo is not well aligned with the cards as in the Zeplin design. Check screenshots below:
- Zeplin design
![image](https://user-images.githubusercontent.com/62849872/223979625-35baa0e4-2c8c-4615-a05f-9cb5a1822d04.png)

- Your design
![image](https://user-images.githubusercontent.com/62849872/223979996-27264d66-9249-4295-9522-19772b9a4676.png)